src/components/page-pipeline-jobs/application-alerting.js

```diff
@@ -86,26 +86,24 @@ const ApplicationAlerting = ({
       {alertingConfig && (
         <>
           <Typography>
-            <Icon
-              data={alertingConfig?.enabled ? notifications : notifications_off}
-            ></Icon>
             Alerting is{' '}
             <Typography as="span" bold>
               {alertingConfig?.enabled ? 'enabled' : 'disabled'}
-            </Typography>
-            <Button
-              style={{ marginLeft: '16px' }}
-              variant="outlined"
-              onClick={() => setVisibleScrim(true)}
-            >
-              Configure
+            </Typography>{' '}
+            <Button variant="ghost" onClick={() => setVisibleScrim(true)}>
+              Setup alert
```

Comment:
When alerting is already setup, say something like `Edit alert`?
```suggestion
              {alertingConfig?.enabled ? 'Edit alert' : 'Setup alert'}
```

Reply:
Fixed